src/utilities/remotesd.c

```diff
@@ -222,6 +255,19 @@ void rle_finalise(void)
   }
 }
 
+void wait_for_sdcard_to_go_busy(void)
+{
+  if (xemu_flag) {
+    // pause a little here, to permit xemu to echo the last command back to mega65_ftp
```

Comment:
add a comment here that xemu completes SD card jobs instantaneously, and thus never goes busy

Reply:
Ok, comment updated